charts/posthog/values.yaml

```diff
@@ -334,24 +334,32 @@ postgresql:
   nameOverride: posthog-postgresql
   # -- Postgresql database name
   postgresqlDatabase: posthog
-  # -- Postgresql database username
-  postgresqlUsername: postgres
   # -- Postgresql database password
   postgresqlPassword: postgres
   persistence:
     # -- Enable persistence using PVC
     enabled: true
     # -- PVC Storage Request for PostgreSQL volume
     size: 10Gi
-  # -- Host postgres is accessible from. Only set when internal PG is disabled
+  # -- Secret containing desired postgresql password. Secret must be behind key `postgresql-password`.  When defined the `postgresqlPassword` field is ignored
+  existingSecret:
+
+externalPostgresql:
+  # -- Host postgresql is accessible from
   postgresqlHost:
```

Comment:
nit: I would use host, port, database, username, password

Reply:
There's a trade-off here - do we want to keep the keys consistent with other services or consistent with `postgresql.`? 

I'd argue for `posgresql.` but ymmv.

Reply:
I think I'd lean on the side of mirroring `postgresql.`, but I can see it might be surprising to see a difference to the other configs.